lib/active_merchant/billing/gateways/digital_river.rb

```diff
@@ -69,6 +69,27 @@ def purchase(options)
         end
       end
 
+      def refund(money, _transaction_id, options)
+        currency = options[:currency] || currency(money)
+        params =
+          {
+            'order_id' => options[:order_id],
+            'currency' => currency.upcase,
+            'amount' => localized_amount(money, currency).to_f,
```

Comment:
Is calling `.to_f` here is really necessary?
I don't see it in any other place in ActiveMerchant 🤔

Reply:
DR expects a float and `localized_amount) returns a string, so I followed Square doing `.to_i`: https://github.com/chargify/active_merchant/blob/master/lib/active_merchant/billing/gateways/square.rb#L187

I guess you're worried about float precision and all that? I think it's fine, it's the last point before sending the value to DR, really. 🤔

Reply:
I've thought that probably DR gem requires that format after I had written this comment 😉